zhaquirks/tuya/ts0601_sensor.py

```diff
@@ -75,6 +75,34 @@ class TemperatureHumidityManufCluster(TuyaMCUCluster):
     }
 
 
+class TemperatureHumidityBatteryStatesManufCluster(TuyaMCUCluster):
+    """Tuya Manufacturer Cluster with Temperature and Humidity data points. Battery states 25, 50 and 100%."""
+
+    dp_to_attribute: Dict[int, DPToAttributeMapping] = {
+        1: DPToAttributeMapping(
+            TuyaTemperatureMeasurement.ep_attribute,
+            "measured_value",
+            converter=lambda x: x * 10,  # decidegree to centidegree
+        ),
+        2: DPToAttributeMapping(
+            TuyaRelativeHumidity.ep_attribute,
+            "measured_value",
+            # converter=lambda x: x * 10,  --> move conversion to TuyaRelativeHumidity cluster
+        ),
```

Comment:
I wonder if we can get the `DPToAttributeMapping` for `1` and `2` from the `TemperatureHumidityManufCluster`, so we don't duplicate that code. I'm not yet sure if it's a great idea though.

Maybe something like the following could work? (didn't test)
```python
        1: TemperatureHumidityManufCluster.dp_to_attribute[1],
        2: TemperatureHumidityManufCluster.dp_to_attribute[2],
```
What do you think about this?

(and thanks for the PR!)

Reply:
Thanks for the feedback. I tested it; the code works.